From: =?utf-8?q?F=C3=A9lix_Sipma?= <felix.sipma@no-log.org>
Date: Thu, 20 Oct 2016 15:09:42 +0200
Subject: Debianize README.md

- remove installation instructions and CI links
- modify link to screenshot file

Forwarded: not-needed
---
 README.md | 36 +-----------------------------------
 1 file changed, 1 insertion(+), 35 deletions(-)

diff --git a/README.md b/README.md
index 3cd2820..3dfcdf6 100644
--- a/README.md
+++ b/README.md
@@ -1,8 +1,6 @@
 patat
 =====
 
-[![Build Status](https://img.shields.io/travis/jaspervdj/patat.svg)](https://travis-ci.org/jaspervdj/patat) [![Hackage](https://img.shields.io/hackage/v/patat.svg)](https://hackage.haskell.org/package/patat) [![GitHub tag](https://img.shields.io/github/tag/jaspervdj/patat.svg)]()
-
 `patat` (**P**resentations **A**top **T**he **A**NSI **T**erminal) is a small
 tool that allows you to show presentations using only an ANSI terminal.  It does
 not require `ncurses`.
@@ -20,7 +18,7 @@ Features:
   syntax files.
 - Written in [Haskell].
 
-![screenshot](extra/screenshot.png?raw=true)
+![screenshot](examples/screenshot.png?raw=true)
 
 [Kate]: https://kate-editor.org/
 [Haskell]: http://haskell.org/
@@ -29,9 +27,6 @@ Features:
 Table of Contents
 -----------------
 
--   [Installation](#installation)
-    -   [Using stack](#using-stack)
-    -   [Using cabal](#using-cabal)
 -   [Running](#running)
 -   [Input format](#input-format)
 -   [Configuration](#configuration)
@@ -39,35 +34,6 @@ Table of Contents
     -   [Syntax Highlighting](#syntax-highlighting)
 -   [Trivia](#trivia)
 
-Installation
-------------
-
-You can build from source using `stack install` or `cabal install`.  `patat` is
-also available from [Hackage].
-
-[Hackage]: https://hackage.haskell.org/package/patat
-
-For people unfamiliar with the Haskell ecosystem, this means you can do either
-of the following:
-
-### Using stack
-
-1. Install [stack] for your platform.
-2. Clone this repository.
-3. Run `stack setup` (if you're running stack for the first time) and
-   `stack install`.
-4. Make sure `$HOME/.local/bin` is in your `$PATH`.
-
-[stack]: https://docs.haskellstack.org/en/stable/README/
-
-### Using cabal
-
-1. Install [cabal] for your platform.
-2. Run `cabal install patat`.
-3. Make sure `$HOME/.cabal/bin` is in your `$PATH`.
-
-[cabal]: https://www.haskell.org/cabal/
-
 Running
 -------
 
